designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRender.kt

```diff
@@ -162,13 +167,40 @@ internal fun Modifier.squooshRender(
                         val nodeSize =
                             Size(computedLayout.width * density, computedLayout.height * density)
 
+                        // Check for an image customization with context. If it exists, call the
+                        // custom image function
+                        // and provide it with the frame's background and size.
+                        val customImageWithContext =
```

Comment:
nit: move them inside the `squooshShapeRender` where image customization happens?

Reply:
good point, I think I thought that function was shared with non-squoosh. Fixed now.